docs/adguard-for-android/solving-problems/multiple-user-profiles.md

```diff
@@ -9,9 +9,23 @@ This article covers AdGuard for Android, a multifunctional ad blocker that prote
 
 :::
 
+In this article you will find the methods on how to solve issues with AdGuard when you set up *multiple accounts* or *Restricted Profile*s on your android devices.
+
+## Problems caused by multiple user profiles
+
+On Android 9 and later, if AdGuard is installed for more than one user profile on your device, you may encounter problems when uninstalling the app. When you uninstall AdGuard from one user profile, the app will still appear on the app list, but you won't be able to uninstall or reinstall it. It happens because AdGuard is installed for a different user profile on the device.
+
+If you try to reinstall AdGuard after an unsuccessful removal attempt, you will see the error message "You can't install the app on your device".
+
+To solve this problem, you need to uninstall the application for all users: go to Settings → All apps → AdGuard. Tap the three dots at the upper right corner and select *Uninstall for all users*.
```

Comment:
Tap the three-dot menu in the top right corner and select *Uninstall for all users*.